src/App.vue

```diff
@@ -1,27 +1,51 @@
 <template>
-  <div id="app">
-    <Users></Users>
-  </div>
+    <div id="app">
+        <template v-if="authorizedUser">
+            <v-btn depressed small @click="signOut">Sign Out</v-btn>
+            <Users></Users>
+        </template>
+        <template v-else>
+            Sign in to see user list
+            <SignIn class="m-auto" @onSignIn="onSignIn"></SignIn>
+        </template>
+    </div>
 </template>
 
 <script>
-import Users from './components/Users'
+  import { googleAPI } from '@/services/gapi';
+  import SignIn from '@/components/SignIn';
+  import Users from '@/components/Users/Users';
 
-export default {
-  name: 'app',
-  components: {
-    Users
-  }
-}
+  export default {
+    name: 'app',
+    methods: {
```

Comment:
please use correct order in components 
for example rules from esLint 
https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/order-in-components.md